### ISSUE TITLE
Fix App Cache for coding applications

### ISSUE THREAD

**Issue by colinfizgig** [argonjs/argon#35]
Right now when you update the app.js or index.html or any other file that loads as part of an argon application, it won't show the update from the webserver, since the file gets stored in the ios app cache.  this should be fixed.  Alternatively if its possible to fix this on the webserver side that work around should be documented.

**Comment by blairmacintyre** [comment]
This isn't an argonjs/argon issue, if it's a bug in the browser.  Can you verify which of the above it is?  Please see if a website that "reloads correctly" (when the html or .js files change) in mobile safari on iOS does not reload correctly in Argon4?  If Argon4 is using the cache when it shouldn't, please submit this as an issue on the argon-app repo.

If it behaves the same as safari, can you figure out what's wrong with your server?  A server should serve up the file timestamps, and the browser should reload if they've changed ... but I think that depends (somehow) on the server setup, what it returns in the headers, how long it says the resource is valid for, etc.  We may need to add a way to tell Argon4 to dump it's cache (like Safari has in it's "develop" menu).  Again, if this is needed, we need to know.  

In summary, I'm not sure what the right solution is, so it would be great if you (or someone) can figure out what the facts are and provide a solid suggestion.
